.eslintrc

```diff
@@ -5,12 +5,23 @@
     "browser": true,
     "es6": true
   },
-  "parser": "babel-eslint",
-  "ecmaFeatures": {
-    "modules": true,
-    "jsx": true
-  },
   "rules": {
+    "comma-dangle": [2, "never"],
     "semi": [2, "never"],
-  }
+    "arrow-body-style": [2, "always"],
+    "class-methods-use-this": 0,
+    "no-console": 0,
+    "no-param-reassign": 0,
+    "func-names": 0,
+    "no-restricted-syntax": 0,
+    "no-plusplus": 0,
+    "no-continue": 0,
+    "no-prototype-builtins": 0,
+    "prefer-rest-params": 0,
+    "no-unused-vars": [2, { "args": "none" }],
+    "no-underscore-dangle": 0,
+    "react/jsx-filename-extension": 0,
+    "react/forbid-prop-types": 0,
+  },
+  "parser": "babel-eslint"
```

Comment:
don't think we need this, but we may need it for some stage-0 features. I'd happily .eslintignore anything that is stage0 though, since it's supposed to be experimental